### ISSUE TITLE
Weird and annoying snapping behaviour (Modular Mixer)

### ISSUE THREAD

**Issue by Lykdraft** [kmatheussen/radium#1291]
Hey there,

updated from 5.98 to the latest version and I noticed some weird
snapping behaviour in the modular mixer window.
Sometimes the modules just snap to a field/gridslot besides/around 
the main pipe?

![snapping3](https://user-images.githubusercontent.com/18437561/96045460-444eaf00-0e72-11eb-9970-5d6b74a1b773.gif)

**Comment by kmatheussen** [comment]
I can't reproduce it. Maybe it's a mac thing only.

**Comment by Lykdraft** [comment]
I hope I can check the windows version this weekend.
It is still present in the latest mac version however.

**Comment by kmatheussen** [comment]
Yeah, I can usually reproduce this behavior. Strange.

**Comment by kmatheussen** [comment]
(I can reproduce it on a mac, I mean. The problem doesn't seem to be present on other platforms)

**Comment by kmatheussen** [comment]
It might be a coincidence, but I've tried a little bit with Radium 6.7.68 now, and the bug seems gone.

**Comment by Lykdraft** [comment]
Yeah, same on my end. Just updated and it seems to be gone. Fingers crossed. This was **really** annoying.. :)

**Comment by kmatheussen** [comment]
I guess updating Qt fixed it. One week of work payed off. :-)
